bravado/http_future.py

```diff
@@ -132,6 +133,9 @@ def response(self, timeout=None, fallback_result=None, exceptions_to_catch=FALLB
         incoming_response = None
         exc_info = None
         request_end_time = None
+        if self.request_config.force_fallback_result:
+            exceptions_to_catch = tuple(chain(exceptions_to_catch, (ForcedFallbackResultError,)))
```

Comment:
Is this needed? Could we just override `exceptions_to_catch` with `ForcedFallbackResultError`

Reply:
Like discussed offline: Let's stick to the approach of only throwing the `ForcedFallbackResultError` in case no other exception is thrown, as that makes it significantly easier to handle cleanly for async HTTP clients. I've moved it to the end of the block.